doozerlib/cli/get_nightlies.py

```diff
@@ -104,17 +109,24 @@ async def get_nightlies(runtime, matching: List[str], exclude_arch: List[str], a
     ])
 
     # find sets of nightlies where all arches have equivalent content
-    nightly_sets = []
-    for nightly_set in generate_nightly_sets(nightlies_for_arch):
+    nightly_sets = generate_nightly_sets(nightlies_for_arch)
+    consistent_nightly_sets = []
+    for nightly_set in nightly_sets:
         # check for deeper equivalence
         await nightly_set.populate_nightly_content(runtime)
         if nightly_set.deeper_equivalence():
-            nightly_sets.append(nightly_set)
+            consistent_nightly_sets.append(nightly_set)
             util.green_print(nightly_set.details() if details else nightly_set)
-            if len(nightly_sets) >= limit:
+            if len(consistent_nightly_sets) >= limit:
                 break  # don't spend time checking more than were requested
 
-    if not nightly_sets:
+    remaining = limit - len(consistent_nightly_sets)
+    if allow_inconsistency and remaining > 0:
+        for nightly_set in nightly_sets[:remaining]:
+            util.yellow_print(nightly_set.details() if details else nightly_set)
```

Comment:
one minor nit with this is that `nightly_sets` includes `consistent_nightly_sets` so consistent ones will be double-printed if the limit is high enough. We're unlikely to run this with a limit higher than 1 or 2 so it's not exactly important but I just like to be thorough. Adding a commit that I think simplifies this.